views/layout.hbs

```diff
@@ -40,7 +40,7 @@
 
     <div class="footer">
         <p>&copy; ChessHub 2014, <i class="fa fa-code"></i> with <i class="fa fa-heart"></i> by
-            <a target="_blank" href="http://www.mahmoud-benhassine.fr">Mahmoud Ben Hassine</a></p>
+            <a target="_blank" href="http://www.mahmoudbenhassine.com">Mahmoud Ben Hassine</a></p>
```

Comment:
This is outdated comparing to master
(Should be mahmoud-benhassine.fr, see 7f4b96a4641c8f01c67916c57cb7e0cde5c2bde9)